packages/esm-service-queues-app/src/queue-table/cells/queue-table-action-cell.component.tsx

```diff
@@ -61,7 +61,7 @@ export function QueueTableActionCell({ queueEntry }: QueueTableCellComponentProp
 }
 
 export const queueTableActionColumn: QueueTableColumn = {
-  headerI18nKey: '',
+  header: translateFrom('@openmrs/esm-service-queues-app', 'actions', 'Actions'),
```

Comment:
Same comment as above, just use the `t` function (this goes for all of the other similar changes to other files in this PR, I'm not going to keep making the same comment).  Or if this needs to be like this, can you describe why?